src/unless.js

```diff
@@ -0,0 +1,31 @@
+var _curry2 = require('./internal/_curry2');
+var identity = require('./identity');
+var ifElse = require('./ifElse');
+
+
+/**
+ * Creates a function that will first pass its argument to the given predicate
+ * function. If the predicate returns false, the function will return the result
+ * of calling the `whenFalse` function with the same argument. If the predicate
+ * returns true, the argument is returned as is.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @see R.ifElse, R.when
+ * @sig (a -> Boolean) -> (a -> a) -> (a -> a)
+ * @param {Function} condition A predicate function
+ * @param {Function} whenFalse A function to invoke when the `condition`
+ *                             evaluates to a falsy value.
+ * @return {Function} A new function that will process the `whenFalse` function
+ *                    depending upon the result of the `condition` predicate.
+ * @example
+ *
+ *      // coerceArray :: (a|[a]) -> [a]
+ *      var coerceArray = R.unless(R.isArrayLike, R.of);
+ *      coerceArray([1, 2, 3]); //=> [1, 2, 3]
+ *      coerceArray(1);         //=> [1]
+ */
+module.exports = _curry2(function(pred, whenFalseFn, x) {
+  return ifElse(pred, identity, whenFalseFn, x);
+});
```

Comment:
We like to use named function expressions rather than anonymous function expressions at the top level.

Also, what is the `x` doing?